Formula/wireguard-go.rb

```diff
@@ -0,0 +1,17 @@
+class WireguardGo < Formula
+  desc "Userspace Go implementation of WireGuard"
+  homepage "https://www.wireguard.com/"
+  url "https://git.zx2c4.com/wireguard-go/snapshot/wireguard-go-0.0.20180514.tar.xz"
+  sha256 "7b26d66fe9b5704856d7d30076928d5a56cc42a271af2a95dc43826fcad213b6"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    system "make", "PREFIX=#{prefix}", "install"
+  end
+
+  test do
+    system "#{bin}/wireguard-go"
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
This isn't possible at the moment, as the app has an intentionally minimal CLI interface, following a predefined spec, but I can modify the test to parse some error-output so that at least there's _something_.